ms/Log.java

```diff
@@ -0,0 +1,56 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+
+public class Log
+{
+
+    public FileWriter createLogWriter()
+    {
+        FileWriter fileWriter = null;
+        try
+        {
+            //creating the log file
+            File myObj = new File("/logs/logs.txt");
+            if (myObj.createNewFile()) {
```

Comment:
`createNewFile` would need exception handling. https://docs.oracle.com/javase/7/docs/api/java/io/File.html#createNewFile()

Reply:
So there is already a try catch block around that. Can we talk more on this ?